PromoExchange/batman
# Less than minimum support in prebid

See https://trello.com/c/biYPVA7d
### Test Plan

@spencerapplegate - Should update the Pavia files directly on AWS or is there a different strategy for preview apps.
1. Load Pavia with updated data into preview app
2. Purchase 150 of SM-2428
3. Ensure purchase price is XXXX
4. Ensure prebid log includes reference to less than minimum surcharge.
### Notes

There is no change required for the product loaders.
The migration should run to add the new upcharge type (less than minimum)
